### PR TITLE
Import National Park Service data as campus records

### PR DESCRIPTION
**Pull request by stepps00**
Related to https://github.com/whosonfirst-data/whosonfirst-data/issues/1699.

This PR imports new `campus` records for places in the National Park Service's Parks boundary file. There are 419 new records, each representing a National Park, National Historic Site, National Memorial, or similar designation.

Mapshaper-based centroids, various concordances, name translations, and hierarchies are included. This will probably need some fine-tuning over time (especially untangling various WOF venues from these places), but overall merge-able now.

No PIP required, can merge once approved.

### CODE REVIEW

**Comment by stepps00** [comment]
Paging @nvkelso - I've had this PR open for a bit. It includes new `campus` records for parks (all National Parks included), historic sites, and memorials managed by the National Park Service. I'd like to get this PR in, then add additional `campus` records for other, similar places like National Forests, BLM lands, etc. Thoughts about importing / changes that are needed to these records?

File: data/147/772/729/3/1477727293.geojson [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    "geom:longitude":-122.687716,
+    "iso:country":"US",
+    "lbl:latitude":48.211316,
+    "lbl:longitude":-122.674423,
+    "mps:latitude":48.211316,
+    "mps:longitude":-122.674423,
+    "mz:hierarchy_label":1,
+    "mz:is_current":1,
+    "mz:is_landuse_aoi":1,
+    "name:ceb_x_preferred":[
```
Comment (nvkelso) [inline data/147/772/729/3/1477727293.geojson:20]: We should consider blocking these `ceb` names from Wikidata, they continue to have a bad bot over there.

**Review by nvkelso — COMMENTED** [review]
I'm generally fine with this. It lines up with some of the other "neighbourhoods" we already x-list as campuses.

Note there is overlap with venues for the same.

File: data/147/772/729/3/1477727293.geojson (hunk 1 above)
Comment (stepps00) [reply data/147/772/729/3/1477727293.geojson:20]: I'm going to track this independent of this PR, as both the `ceb` and `vol` language tags are suspect.
I'll likely merge as-is, then iterate over all admin records to remove/rework those name properties.

**Comment by stepps00** [comment]
This PR is in a good spot, caveat being that there is more work to do around matching these to existing venues that represent some of these places (flagging as coterminous?). Merge-able once the linked sources PR is merged.

**Pull request merged**